### ISSUE TITLE
nexus3 repository create --cleanup policy fails: java.lang.String cannot be cast to java.util.Set

### ISSUE THREAD

**Issue by manics** [thiagofigueiro/nexus3-cli#77]
Connecting to Docker `sonatype/nexus3:3.19.1`
```
$ nexus3 --version
2.1.0

$ nexus3 cleanup_policy list
Name          Format        lastDownloaded   lastBlobUpdated
test          ALL_FORMATS   null             1209600

$ nexus3 repository create hosted maven maven-internal --cleanup=test
Traceback (most recent call last):
  File "/venv/bin/nexus3", line 11, in <module>
    sys.exit(main())
  File "/venv/lib64/python3.6/site-packages/nexuscli/cli/__init__.py", line 108, in main
    return _run_subcommand(arguments, maybe_subcommand)
  File "/venv/lib64/python3.6/site-packages/nexuscli/cli/__init__.py", line 90, in _run_subcommand
    return subcommand_method(argv)
  File "/venv/lib64/python3.6/site-packages/nexuscli/cli/subcommand_repository.py", line 154, in main
    return command_method(util.get_client(), arguments)
  File "/venv/lib64/python3.6/site-packages/nexuscli/cli/subcommand_repository.py", line 117, in cmd_create
    nexus_client.repositories.create(r)
  File "/venv/lib64/python3.6/site-packages/nexuscli/api/repository/collection.py", line 248, in create
    raise exception.NexusClientCreateRepositoryError(resp)
nexuscli.exception.NexusClientCreateRepositoryError: {'name': 'nexus3-cli-repository-create', 'result': 'java.lang.ClassCastException: java.lang.String cannot be cast to java.util.Set'}
```

**Comment by bt-thiago** [comment]
This bug was likely caused by this change in Nexus 3.19.0: https://issues.sonatype.org/browse/NEXUS-19525

**Comment by thiagofigueiro** [comment]
This fix is now available in version 2.1.1.

It was a bit annoying that Sonatype broke the groovy API by changing the parameter type from String to Set. It would be appreciated if they could keep backwards-compatibility since the REST API doesn't do much and groovy scripts are the only way to do most operations.

**Comment by manics** [comment]
Thanks for the quick fix! I agree with you, I didn't expect Sonatype to make a breaking change like this.
